src/Chapter2.hs

```diff
@@ -740,8 +771,13 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
-
+contains :: Int -> [[Int]] -> [[Int]]
+-- contains a xs = filter (not . null) (map (go a) xs)
+--   where
+--     go :: Int -> [Int] -> [Int]
+--     go a l = 
+--       if a `elem` l then l else []
+contains a xs = filter (elem a) xs
```

Comment:
Yes, this is an awesome and short solution!
You could also notice that you can do eta-reduction on this function as well 😉